DEVELOPMENT.rst

```diff
@@ -0,0 +1,39 @@
+Developing on edx-sga
+==============================
```

Comment:
nit: strict .RST rules require that the formatting line is the same length as the text it formats.